packages/airnode-validator/src/config/config.ts

```diff
@@ -52,7 +52,7 @@ export const providerSchema = z
 
 export const amountSchema = z
   .object({
-    value: z.number(),
+    value: z.number().lte(9007199254740991), // 2**53 - 1
```

Comment:
```suggestion
    value: z.number().lte(2**53 - 1),
```

Reply:
I chose to hardcode it in order to avoid 2**53 since that cannot be accurately represented as an integer.

Reply:
Yeah, that's a good point.